.github/workflows/build_windows_release_candidate.yml

```diff
@@ -0,0 +1,32 @@
+name: Build Windows Unsigned Release Candidate
+
+on: [workflow_dispatch, workflow_call]
+
+jobs:
+  windows_release_build:
+    name: Build and Pack Windows Unsigned Release Binaries
+    runs-on: windows-latest
+    timeout-minutes: 10
+    env:
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+
+      - name: Install NPM Dependencies
+        run: npm ci
+
+      - name: Build Windows Release Binaries
```

Comment:
```suggestion
      - name: Build Windows Installer
```

Reply:
I think we don't need to say "Release" here, since we are in the context of the release.